src/UdpDataProtocol.cpp

```diff
@@ -478,7 +485,13 @@ void UdpDataProtocol::run()
                   << " before setRealtimeProcessPriority()" << std::endl;
     //std::cout << "Experimental version -- not using setRealtimeProcessPriority()" << std::endl;
     // Anton Runov: making setRealtimeProcessPriority optional
-    if (mUseRtPriority) { setRealtimeProcessPriority(); }
+    if (mUseRtPriority) {
+#if defined (__MAC_OSX__)
```

Comment:
It would be nice to have less preprocessor macros. Given how many of them there are in UdpDataProtocal, this might be a bigger change post this merge.